jobs/candlepinPerformanceTestMaintenance.groovy

```diff
@@ -0,0 +1,38 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/CandlepinPerformanceMaintanance") {
+    disabled()
+    description('This job runs candlepin maintenance of performance tests')
```

Comment:
Let's say "This job updates the performance test baseline data."

Reply:
OK